### ISSUE TITLE
Download non-English chapters

### ISSUE THREAD

**Issue by Deathspike** [Deathspike/mangarack#42]
> Might be a good idea to have an option to allow Mangarack to download stuff in other language than English?

Source: https://github.com/Deathspike/mangarack/issues/41

**Comment by Deathspike** [comment]
Bato.to only. Proposed:

`----component.core.batoto.language spanish`

As an example

**Comment by XellossMetallium** [comment]
Personally I'm not particullary interested.. italian scanlators are bad in their own language.. I find it easier for my mental sanity to read in english. :D
That being said.. the only improvement I can think of could be a command used for setting a "fallback" language or a "second" language. 
Possible uses:
- When a series is missing in english it could be downloaded i.e. in spanish
- In the (unlikely) case a series is available in english up to chapter 30 but in spanish up to ch 36 chapters 31-36 could be downloaded in spanish (with a proper tag attached)
- In the above case when 31-36 become available in english mangarack could overwrite the spanish ones.
Maybe this could work better as a different command, since the purpose would be to accomodate bilingual readers more than opening mangarack to non-english speakers.
As always I don't know how much of this is technically possible, but if it is it could be very cool.

**Comment by Deathspike** [comment]
In the philosophy of predictable behavior (I keep throwing this point around, sorry about that) it would be a bad idea to download chapters depending on whether or not chapters are available in another language. This would mean that the chapter listing is time dependent and other language chapters would disappear as English ones become available.

Rethinking my earlier proposition, which implies that you could hide English-language chapters, would potentially cause confusion as well as it would be completely capable of removing an entire chapter listing. Especially for the more novice user, this could be a big problem, as there would be many series that simply do not have chapters -- leading to the conclusion that MangaRack is a broken POS.

Perhaps it would be better to have `--component.core.batoto.additionalLanguages` that can add one or more languages to be allowed in the chapter listing. This, of course, implies that you will always get English results plus the language you've configured, regardless of whether or not you can read English in the first place?

Thoughts? @XellossMetallium @Mereck13579 @Mike43110

**Comment by XellossMetallium** [comment]
I see your point, additionalLanguages can be a good compromise between simplicity and "covering the holes".. but like you pointed out the consequence would be having a (possibly) useless language downloaded.. and since that language would be english it would probably be, in size, the bigger part of the library. 

What I mean is that while "additionalLanguages" adresses well enough what I suggested (with the only minus of having some duplicates, but who cares), to the user who does not read eng the "exclude" would still be better, since he will probably gain a lot of disk space and he will be able to add a series still not translated to his list in advance, having it downloaded only when the translation comes out in a language he can actually read.*

In short I would go with both commands, since they adress different (albeit similar) cases: one for bilingual readers and one for the non english readers, with a good note in the readme about the consequence of excluding the main language (little or no results).

*about this.. something like the old "ignore errors" would be necessary, I think.

**Comment by Mereck13579** [comment]
I agree with @XellossMetallium, having both option is quite good if they are adequately documented (though i'll probably only use English anyway)

**Comment by Deathspike** [comment]
https://github.com/Deathspike/mangarack/issues/79
